org2jekyll.el

```diff
@@ -143,8 +162,9 @@ The `'%s`' will be replaced respectively by name, the author, the generated date
   "Generate a formatted now date."
   (format-time-string "%Y-%m-%d %a %H:%M"))
 
-(defun org2jekyll-default-headers-template (blog-layout blog-author post-date post-title post-description post-tags post-categories)
+(defun org2jekyll-default-headers-template (blog-type blog-layout blog-author post-date post-title post-description post-tags post-categories)
   "Compute default headers.
+BLOG-TYPE is the type of the post, linked with org porjects.
```

Comment:
TYPE here is just the COLLECTION.

From my understanding of collections, it's a way for users to define a named set of articles.
By default there are two implicit collections: pages and (ordinary) posts. Users can define collections like: wikis, articles-of-some-fun-project and etc.

These collections are dedfined in Jekyll configurations and usually articles in a collection has its own sub-directory (instead of _posts), maybe even its own layout and etc. To support publishing articles to different collections, we can define one org publish project for each collection, like what we do for pages and posts.